library/GitHub/Workflow/Command/Message.hs

```diff
@@ -0,0 +1,27 @@
+module GitHub.Workflow.Command.Message
+  ( Message
+  , toByteStringBuilder
+  ) where
+
+import Control.Category
+import Data.ByteString.Builder qualified as BSB
+import Data.String (IsString)
+import Data.Text (Text)
+import Data.Text qualified as T
+import Data.Text.Encoding qualified as T
+import Prelude (Eq, Ord, Show)
+
+newtype Message = Message {text :: Text}
+  deriving newtype (Eq, Ord, Show, IsString)
+
+toByteStringBuilder :: Message -> BSB.Builder
+toByteStringBuilder =
+  (.text)
+    >>> T.concatMap
```

Comment:
Similar note here: using `(>>>)` just seems unmotivated when these could be straight forward `(.)` expressions any Haskell 101-er would recognize.

Reply:
:shrug: They're the same function

Reply:
Well, sure but `(.)` is familiar and in `Prelude` (and order-swapped, for better or worse).

I get what you're going for, you think there should just be the one more general thing and that's what we should all use. So you're "being the change you want to see" here. I agree with that on principle, I just think there are competing practical concerns when part of a team that's trying to be inclusive of more junior Haskellers that will have been exposed to `(.)` far more in the early stages of learning. And who are having so much new things thrown at them that I just don't see the extra import and new name (even if more general, this one is still the "new" one) being anything other than a distracting hurdle.

On the other hand, maybe we aren't actually prioritizing writing "welcoming" Haskell in that way, that could be a fair point. I'd be curious what the team thinks.